client/cypress/end-to-end/ballot-comparison-failure-cases.spec.js

```diff
@@ -217,6 +217,13 @@ describe('Ballot Comparison Failure Cases', () => {
     cy.loginAuditAdmin(auditAdmin)
     cy.findByText(`TestAudit${id}`).click()
     cy.findByText('Review & Launch').click()
+  
+    // custom size validation test
+    cy.findByText('Enter your own sample size (not recommended)').click()
+    cy.findByRole('spinbutton').type('30').blur()
+    cy.findByText('Must be less than or equal to: 15 (the total number of ballots in the contest)')
```

Comment:
this is frontend validation, right? if so, i think this would be better tested in a unit test, not an end-to-end test, since it doesn't involve communication between front and backend